content/en/docs/setup/upgrade/index.md

```diff
@@ -108,6 +108,10 @@ If the old control plane does not have a revision label, uninstall it using its
 $ istioctl x uninstall -f manifests/profiles/default.yaml
 {{< /text >}}
 
+{{< warning >}}
+Note that this command would also remove gateway deployments, use it with caution. The issue would be fixed in release `1.7.1`
```

Comment:
```suggestion
This command will also remove gateway deployments, which will then need to be redeployed. The issue will be fixed in release `1.7.1`
```

Reply:
@richardwxn if the fix is NOT in 1.7.0, can we just recommend users how to clean up manually without uninstall cmd?

Using uninstall cmd would create a non working cluster for users without the fix.

Reply:
@linsun the use cases of --revision and --purge works, which I expect most people would use, I can remove the part of -f, wdyt?

Reply:
Looks like the fix will land in 1.7.0 afterall.  If --revision works for default, I agree with removing the -f portion.

Reply:
updated the description